sources-dist.json

```diff
@@ -1622,6 +1622,11 @@
     "icon": "https://raw.githubusercontent.com/iobroker-community-adapters/ioBroker.ring/master/admin/ring.png",
     "type": "hardware"
   },
+  "rmb-bhkw": {
+    "meta": "https://raw.githubusercontent.com/satchafunkilus/ioBroker.rmb-bhkw/master/io-package.json",
+    "icon": "https://raw.githubusercontent.com/satchafunkilus/ioBroker.rmb-bhkw/main/admin/rmb-bhkw.png",
```

Comment:
@satchafunkilus 
Path is wrong here. You need to adjust the pull request to the repositories repo so that it points to the correct icon location: https://raw.githubusercontent.com/satchafunkilus/ioBroker.rmb-bhkw/master/admin/rmb-bhkw.png (not main).

Just commit into the same branch again and that should update the pull request.

Reply:
@Garfonso thanks for the hint, done!